examples/websocket-proxy-server/websocket_proxy_server.bal

```diff
@@ -82,29 +92,36 @@ service<http:WebSocketClientService> ClientService {
 
         endpoint http:WebSocketListener serverEp =
                         getAssociatedServerEndpoint(caller);
-        serverEp->pushText(text, final = finalFrame)
-            but { error e => log:printError(
-                     "Error occurred when sending text message", err = e) };
+        var err = serverEp->pushText(text, final = finalFrame);
+        if (err is error) {
+            log:printError("Error occurred when sending text message",
```

Comment:
It is necessary to have only upto 80 characters in Ballerina examples so that they will display properly in the website. Hence the file is formatted thus.